build/format-notecards.js

```diff
@@ -0,0 +1,15 @@
+// Remove h4s from any existing notecards and transform them
+// from <div class="note notecard"><h4>Note:</h4>foobar</div> to
+// <div class="note notecard"><p><strong>Note:</strong>foobar</p></div>
+function formatNotecards($) {
+  $(".note.notecard h4").each((_, element) => {
+    const h4 = $(element);
+    const text = h4.text();
+    const p = $("p:first", h4.parents("div.notecard"));
```

Comment:
Does this work? I think I made it up. 
You should try to create  fixture document somewhere in `testing/content/files/en-us/web/...` that contains:
```html
<div class="notecard note">
  <h4>Bla</h4>
  No paragraph here.
</div>
```
and also:

```html
<div class="notecard warning">
  <h4>Ble</h4>
  <p>Paragraph 1</p>
  <p>Paragraph 2</p>
</div>
```

Reply:
It actually worked perfectly 😄 

aside from the h4 not being removed, I tested a few pages and it handled the `<h4>`s correctly. I will create those fixtures and report back with the results.

Reply:
Since we are using `p:first` in the first example, we won't perform the correct transformation. Setting up the fixture document really did help. Once again, thanks for the guidance!

Reply:
Doh, I blundered. I think it is simply going to be "checked" by a test suite with `npm run test:testing`.

Reply:
I might also be completely misunderstanding, and instead of a visible page/URL to arrive at, I need to write the tests with `test()` using jest and then run `npm run test:testing`. That makes more sense.